public/js/index.js

```diff
@@ -0,0 +1,18 @@
+document.addEventListener("DOMContentLoaded", async() => {
```

Comment:
```suggestion
document.addEventListener("DOMContentLoaded", getRoomsList)
```

Reply:
This way it throws an error:
`Uncaught ReferenceError: Cannot access 'getRoomsList' before initialization`

Reply:
That's because the function is defined after it's used. Just move the initialization of listener down

Reply:
Fixed.